### PR TITLE
Use app logo instead of default tkinter logo

### PR DESCRIPTION
**Pull request by ygz213**
This PR adds ability to use logo and deletes logo's `.png` version

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -441,13 +441,13 @@ def show_sidebar():
     ROOT.minsize(180, 90)
 
     # Setting the title of the window.
-    ROOT.title("D-Pad")
+    ROOT.title(" D-Pad")
```
Comment (dhruvnnd) [inline main.py:444]: Why is there blank space before the title? 

> `ROOT.title('_D-Pad')`

**Review by dhruvnnd — CHANGES_REQUESTED** [review]
The icon was there in `.png` format so that we can re-scale it and convert it to `.ico`.

So, I request you not to delete that.

File: main.py (hunk 1 above)
Comment (ygz213) [reply main.py:444]: I thought the window title looks congested.

Comment (dhruvnnd) [reply main.py:444]: > I thought the window title looks congested.

Okay, in that case, I will let it be there.

**Comment by ygz213** [comment]
Can we re-add `.png` version without a commit?

**Comment by dhruvnnd** [comment]
> 
> 
> Can we re-add `.png` version without a commit?

Not sure if we can. 
I'll just add another commit for that.

**Pull request merged**